src/ml.py

```diff
@@ -5,9 +5,19 @@
 import pandas as pd
 from sklearn.preprocessing import StandardScaler
 from sklearn.model_selection import KFold, train_test_split
+from sklearn.linear_model import Lasso
+from sklearn.ensemble import RandomForestClassifier
 
+from typing import Union,Tuple,List,Optional
 
-def train_evaluate_model(x_train, y_train, x_test, y_test, model, verbose):
+def train_evaluate_model(x_train:pd.DataFrame, 
+                         y_train:pd.DataFrame, 
+                         x_test:pd.DataFrame,
+                         y_test:pd.DataFrame, 
+                         model:Union[Lasso,RandomForestClassifier], 
```

Comment:
Is there a way to specify it in a more general way? Lasso and RandomForestClassifier are not the only models supported (although they are the only ones in the example and in the tests, for the moment)

Reply:
![image](https://github.com/apalladi/feature-selection-adding-noise/assets/77753494/39942cb2-b8be-4a94-a881-5d8c677f389f)

We can do like scikit-learn. (Base Estimator)